ServerCore/Pages/Resources/nipd2023/FAQPartial.cshtml

```diff
@@ -0,0 +1,164 @@
+﻿@{
+    var eventPage = Model as ServerCore.ModelBases.EventSpecificPageModel;
+    var Event = eventPage?.Event;
+}
+
+@if (!string.IsNullOrEmpty(Event.FAQContent))
+{
+    @Html.Raw(Event.FAQContent)
+}
+else
+{
+
+    <div class="container">
+        <h2>FAQ for Non-Intern Puzzleday 2023</h2>
+
+        <h3>
+            What is Non-Intern Puzzleday?
+        </h3>
+        <p>
+            The Microsoft Non-Intern Puzzleday is an annual collaborative puzzle competition that challenges teams to solve a large number of original puzzles of all different kinds. These puzzles include wordplay, cryptograms, jigsaws, logic problems, and more.  It is originally created for and played by interns, most of whom have never played in a puzzle event before.  Think of it as a lighter version of Puzzlehunt.
+        </p>
+
+        <h3>
+            Who is NIPD for?
+        </h3>
+        <p>
+            Non-Intern Puzzleday is for full-time Microsoft employees, their family, and friends, but NOT Microsoft interns. It's a team competition open to novice and hardcore puzzle solvers alike. Rule number 1 is &quot;HAVE FUN!&quot; More information about registration is available on the <a asp-page="/EventSpecific/Rules">Rules</a> page.
+        </p>
+
+        <h3>
+            When is NIPD 2023?
+        </h3>
+        <p>
+            NIPD begins at <strong>11 AM PDT on Saturday August 26th, 2023</strong> in Building 99. We'll have a few announcements, introduce the story, and then let solving begin! Solving will end at 5PM, followed by a closing presentation and a chance to go over the answers back in Building 99.
+        </p>
+
+        <h3>
+            How many people can be on a team?
+        </h3>
+        <p>
+            Each team can have a maximum of 6 people.  Out of that 6, at least 2 need to be Microsoft FTEs.  While guests are allowed on campus, remember that you will also need members with Microsoft badges to get into buildings.
```

Comment:
Why has the "no interns" part been removed from this block? Was there a conversation that Dana and I are forgetting?